httpservice/README.md

```diff
@@ -34,53 +34,65 @@ In production:
 
 For local development with hot-reloading:
 
-    docker run -it -v /home/cmi/dev/guardianconnector-change-detection/httpservice/app:/code/app -p 80:80 gccd uvicorn app.app:app --host 0.0.0.0 --port 80 --reload
+    docker run -it -v /home/cmi/dev/guardianconnector-change-detection/httpservice/app:/code/app -p 80:80 --env-file .env gccd uvicorn app.app:app --host 0.0.0.0 --port 80 --reload
 
-## Example API call
+## Payload structure
+
+The `changemaps` endpoint expects a JSON payload consisting of three values, one required and two optional:
+
+* `input_geojson` (array, required): this is a GeoJSON feature collection which will be saved as a temp file and used for the `input_geojson_path` by the GCCD flow.
+* `input_t0` and `input_t1` (string, optional): base64-encoded data strings for the t0 and t1 GeoTIFFs, if provided. 
```

Comment:
How big will these be?  (either raw TIFs or after base64-encoding, which take many more bytes)

I feel like there may be a better way to send both binary data and JSON, but I don't know it off the top of my head.  Will do some thinking/reading and get back to you.  Definitely go forward with your current approach, unless you have a better idea.